panther_analysis_tool/main.py

```diff
@@ -1163,6 +1167,39 @@ def setup_run_tests(  # pylint: disable=too-many-locals,too-many-arguments
     return failed_tests, invalid_specs
 
 
+def validate_packs(analysis_specs: Dict[str, List[Any]]) -> List[Any]:
+
+    invalid_specs = []
+    # first, setup dictionary of id to detection item
+    id_to_detection = {}
+    for analysis_type in analysis_specs:
+        for analysis_spec_filename, _, analysis_spec in analysis_specs[analysis_type]:
+            analysis_id = (
+                analysis_spec.get("PolicyID")
+                or analysis_spec.get("RuleID")
+                or analysis_spec.get("DataModelID")
+                or analysis_spec.get("GlobalID")
+                or analysis_spec.get("PackID")
+                or analysis_spec.get("QueryName")
+                or analysis_spec["LookupName"]
```

Comment:
any reason we aren't using `get` here like all the others?

Reply:
I left the last one as a regular access for the case where we add another analysis type in the future - we would want this to error so we can update the list of possible locations the id can come from